src/scaling/scaling.jl

```diff
@@ -126,6 +126,36 @@ rescale_gradient(r::UnitRange, g) = g
 Implements the chain rule dy/dx = dy/du * du/dx for use when calculating gradients with scaled interpolation objects.
 """ rescale_gradient
 
+@propagate_inbounds function hessian(sitp::ScaledInterpolation{T,N}, xs::Vararg{Number,N}) where {T,N}
+    @boundscheck (checkbounds(Bool, sitp, xs...) || Base.throw_boundserror(sitp, xs))
+    xl = maybe_clamp(sitp.itp, coordslookup(itpflag(sitp.itp), sitp.ranges, xs))
+    h = hessian(sitp.itp, xl...)
+    return symmatrix(rescale_hessian_components(itpflag(sitp.itp), sitp.ranges, Tuple(h), size(h, 1)))
```

Comment:
Do you really need to use `symmatrix` again here? The calculation of the interpolation is expensive (`4^N` where `N` is the dimensionality), but the rescaling is only `N^2`. So you probably don't need to worry much about efficiency here. (The value of `symmatrix` is that you don't have to do redundant computations.)